main.go

```diff
@@ -160,7 +160,25 @@ func main() {
 	logEventPath := viper.GetString("log.path")
 
 	//Create event destinations
-	destinationsService, err := destinations.NewService(ctx, destinationsViper, destinationsStr, logEventPath, syncService, storages.Create)
+	var queryLoggerConfig *logging.Config
+	queryLogsEnabled := false
+	if viper.IsSet("query_log.path") {
```

Comment:
we could do it in the appconfig (there is similar code - global logger initialization)

Reply:
fixed